index.js

```diff
@@ -327,8 +328,7 @@ module.exports = {
   },
 
   postBuild(result) {
-    let appFilePath = stripLeadingSlash(this.app.options.outputPaths.app.js);
-    this._appendAppBoot(result.directory, appFilePath);
+    this._updateAppFilesInManifest(result.directory);
```

Comment:
Is there a way for us to only do this in Embroider land?

Reply:
There's no obvious way and I don't really want to give one to people. Running Embroider and a classic build within the same broccoli system is a valid thing to want to do.

Reply:
@ef4 any reasons for not to provide a way to check the embroider land. It will make it easier for Fastboot's to switch between classic build flow and changes needed to be made for embroider. Also, will help in isolating the build issue specific to embroider.

Reply:
There's really no such thing as "outside" vs "inside" embroider. The addons all get instantiated *before* any embroider code even runs, and that is intentional. Also, the last time we exposed a global flag like this inside broccoli (the one for checking for fastboot) we regretted it, because it's just not really how broccoli works.

So no, I really don't want v1-formatted addons to start adding code to behave special under embroider. The only supported way to do that is to ship as a native V2 addon, after we merge [RFC 507](https://github.com/emberjs/rfcs/pull/507).

Also, this change is an improvement for *all* builds, not just embroider. It moves us in a better architectural direction.

Reply:
Thank you @ef4. This makes sense.